Sources/StreamChatUI/ChatChannelList/ChatChannelListEmptyView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  ChatChannelListEmptyView.swift
+//  StreamChat
+//
+//  Created by Hugo Bernal on 5/07/22.
+//  Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatChannelListEmptyView: _View, ThemeProvider {
+    
+    /// Container which holds all elements except action button in this view.
+    open private(set) lazy var container: ContainerStackView = ContainerStackView().withoutAutoresizingMaskConstraints
+    /// Icon ImageView which is displayed above the title label.
+    open private(set) lazy var iconView: UIImageView = UIImageView().withoutAutoresizingMaskConstraints
+    /// Title label which displays the main message for the empty view.
+    open private(set) lazy var titleLabel: UILabel = UILabel().withoutAutoresizingMaskConstraints
+    /// Subtitle label which displays additional information for the empty view..
+    open private(set) lazy var subtitleLabel: UILabel = UILabel().withoutAutoresizingMaskConstraints
+    /// Button for an additional action on the empty view. E.g. start a new chat.
+    open private(set) lazy var actionButton: UIButton = UIButton(type: .system).withoutAutoresizingMaskConstraints
+    
+    public var actionButtonPressed: (() -> Void)?
+    
+    override open func setUp() {
+        super.setUp()
+        
+        titleLabel.text = L10n.ChannelList.Empty.title
+        subtitleLabel.text = L10n.ChannelList.Empty.subtitle
+        actionButton.setTitle(L10n.ChannelList.Empty.button, for: .normal)
+        actionButton.addTarget(self, action: #selector(didTapActionButton), for: .touchUpInside)
+    }
+    
+    override open func setUpLayout() {
+        super.setUpLayout()
+        
+        directionalLayoutMargins = .init(top: 8, leading: 30, bottom: 8, trailing: 30)
```

Comment:
Maybe 32 instead? Usually we go by 8 multiplies.

Reply:
Updated